src/store/metrics.js

```diff
@@ -34,7 +34,10 @@ export default {
       Vue.delete(state.metrics, metricKey)
     },
 
-    privateSet (state, { metricKey, metric: { name, description, color, traces, marker, errorprone, drawMin, drawMax, drawAvg } }) {
+    privateSet (state, {
+      metricKey,
+      metric: { name, description, color, traces, marker, errorprone, drawMin, drawMax, drawAvg, unit }
```

Comment:
I would change the ordering to `name, description, unit, ...` so that the externally defined properties are grouped together.